apps/aecore/test/aec_tx_pool_tests.erl

```diff
@@ -734,9 +734,7 @@ tx_pool_test_() ->
      ]}.
 
 tx_pool_gc(Height) ->
-    aec_tx_pool_gc:gc(Height),
-    sys:get_status(aec_tx_pool_gc), %% sync point (gc is asynchronous)
```

Comment:
This may leave a harmless redundant call in test - see also https://github.com/aeternity/aeternity/pull/2229